goldmane/pkg/aggregator/aggregator_test.go

```diff
@@ -69,6 +70,22 @@ func ExpectFlowsEqual(t *testing.T, expected, actual *proto.Flow) {
 	}
 }
 
+func aggList(agg *aggregator.LogAggregator, req *proto.FlowListRequest) ([]*proto.FlowResult, error) {
+	var flows []*proto.FlowResult
+	resultChan, err := agg.List(req)
+	if err != nil {
+		return nil, err
+	}
+
+	for result := range resultChan {
```

Comment:
Probably worth verifying the metadata message is correct as well

Reply:
@Brian-McM you're fundamentally adding new functionality that needs to be unit tested though, this isn't just a matter of refactoring it's a matter of asserting the new functionality works as expected.

Reply:
UTs isn't something we should be leaving for a followup PR: we need to be able to trust that merged code is properly tested.  I won't weigh in on what the right UTs are here though.

Reply:
So I did modify some of the UTs, but to be clear the reason why I was saying this was because there are other UTs added to the index / ring index code. Maybe that still wasn't good enough, but I wasn't forgoing testing this.